### PR TITLE
 fix shards

### PR DESCRIPTION
**Pull request by hruljov**

### CODE REVIEW

**Comment by AntonFriberg** [comment]
@AlexeySetevoi Could we perhaps retrigger the tests on this one? They seem related to #71

**Comment by AlexeySetevoi** [comment]
Yes, the failed tests in this PR are associated with #71.
Since #71 changed the test declaration, this PR needs to be rebased to master.
Manual restart in this current PR state - will just fall again.

**Comment by AntonFriberg** [comment]
@AlexeySetevoi This needs approval from you in order to run CI tests

File: README.md
Comment (AntonFriberg) [inline README.md:29]: This looks like a misstake due to merge conflict

File: README.md
Comment (AntonFriberg) [inline README.md:42]: This also looks weird

**Review by AntonFriberg — CHANGES_REQUESTED** [review]
Saw some issues with this that I would guess Alexey would want fixed before he can merge this. Let me know if you have any questions.

**Comment by AntonFriberg** [comment]
@AlexeySetevoi Could you start the CI tests on this one?

**Comment by hruljov** [comment]
@AlexeySetevoi merge?

**Pull request merged**
